src/codec_aom.c

```diff
@@ -715,57 +682,95 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
                 cfg->monochrome = 1;
             }
         }
+    }
+
+    avifBool dimensionChanged = AVIF_FALSE;
```

Comment:
Nit: dimensionChanged => dimensionsChanged

"Dimensions" should be plural.

Reply:
Done